apps/scan/frontend/jest.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
   ],
   coverageThreshold: {
     global: {
-      branches: -54,
+      branches: -55,
```

Comment:
Adding this so we can use this thread to discuss solutions -

I am seeing the branch coverage for `admin_screen` [go down from 100% to 97.91%](https://app.circleci.com/pipelines/github/votingworks/vxsuite/17088/workflows/a492505d-4edc-475c-bda8-2e88ffe5a519/jobs/673529). I think it is because the branch where there is no diagnostic button (because `hardware v3`), then the branch for the `diagnostic_screen` cannot open. So line 244 evaluates to `false/null`, and then line 309 always goes down the same `false` branch.

I was able to fix the other jest errors, but this one has me a little stumped, unless the explanation I gave above is accurate. In that case, this would be an impossible state that would require this bump to the `coverageThreshhold`. Thanks for any advice!

Reply:
`printerStatusQuery.data` can't be undefined on L244 because we check `printerStatusQuery.isSuccess` earlier in the component. So the `?` causes an impossible branch.

A few tips for figuring out coverage issues:
- The terminal coverage report gives precedence to listing uncovered line numbers (last column, in red) before listing uncovered branch line numbers (in yellow). If you mark all of the uncovered lines with ignore comments, then you can see the uncovered branches (in this case, L244).
- Viewing the HTML report in the `coverage` directory is often a quick way to get more info - I think it shows uncovered branches IIRC

Reply:
Ah okay, thanks for the tips, super helpful. 

I was able to see what you mean by commenting out the uncovered line, showing that the L244 was uncovered. That being said, I am a bit surprised that this PR would affect the branch coverage in that case, since it was unrelated to that line of code.